src/drivers/navio_gpio/navio_gpio.cpp

```diff
@@ -0,0 +1,213 @@
+#include <px4_posix.h>
```

Comment:
Do you mean that I should add the PX4 copyright or add include <px4_somefile .h>?
If it is about the copyright, do you need that to all files?
If it is about including some header file, could you please list up what's missing?